microshift_storage/understanding-persistent-storage-microshift.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="understanding-persistent-storage-microshift"]
+= Understanding persistent storage for MicroShift
+include::_attributes/attributes-microshift.adoc[]
+:context: understanding-persistent-storage-microshift
+
+toc::[]
+
+include::modules/storage-persistent-storage-overview.adoc[leveloffset=+1]
+
+include::modules/storage-persistent-storage-lifecycle.adoc[leveloffset=+1]
+
+include::modules/storage-persistent-storage-reclaim-manual.adoc[leveloffset=+2]
+
+include::modules/storage-persistent-storage-reclaim.adoc[leveloffset=+2]
+
+include::modules/storage-persistent-storage-pv.adoc[leveloffset=+1]
```

Comment:
nit: The table "Supported access modes for PVs", which I think is in this file, is not relevant. Let's land the rest of this and use a follow-up PR to remove the table from the MicroShift build.